### ISSUE TITLE
No longer works !

### ISSUE THREAD

**Issue by drogga** [garbarick/SBMega#1]
No longer works as you can't log-in (sign-in), it returns "expired" toast message !

**Comment by drogga** [comment]
@garbarick Do you even care ...?

**Comment by garbarick** [comment]
@drogga, I'v seen it only after your personal appeal.
Application works correct, But it does not support two-factor authentication.

**Comment by drogga** [comment]
@garbarick - No it doesn't work !, did you even tested it before replying ?, because I just did **again** and it's the same issue...
I launch the app, clicked on "Add" (from the context menu), then I type correctly my MEGZ.nz account credentials (E-Mail & Password - I don't use two-factor authentication) and a toast pop-up message appears (shows), saying [-8: Expired], here on Android 4.1.2 Tablet, I even tested it on Android 8 x86 VM Emulator in December, even before I created this "issue", just to be sure that the problem isn't in this particular device\OS ver.

**Comment by drogga** [comment]
@garbarick - No it doesn't work !, I'm sorry, but did you really tested it before replying, claiming that it works ?, because I just did it **again** and it's the same issue...
I launch the app, clicked on "Add" (from the context menu), then I type correctly my MEGA.nz account credentials (E-Mail & Password - I don't use two-factor authentication, but I must admit that the number of characters is long for both) and a toast pop-up message appears (shows), saying **[-8: Expired]**, here on Android 4.1.2 Tablet, I even tested it on Android 8 Emulator back in December, even before I created this "issue", just to be sure that the problem isn't in this particular device\OS ver.

BTW - apologies for bothering you.

**Comment by garbarick** [comment]
@drogga, [-8: Expired] is reply from MEGA. Please, check your account on official application MEGA from google play or from 4pda.

**Comment by drogga** [comment]
I checked it, the official app works, I have v3.4.0 (264) installed and opened it before and after testing your SBMega app, so as I already told you - there's nothing wrong with my account...

**Comment by garbarick** [comment]
I have googled your problem. Posibility your account is second version. For it need another mega#sdk and library. I will try to find decision.

**Comment by drogga** [comment]
Yes, it's a recent one (from ~2 months ago).
OK, please do so...
Thanks in advance ;)

**Comment by drogga** [comment]
@garbarick, Yes, it's a recent one (from ~2 months ago).
OK, please do so...
Thanks in advance ;)

**Comment by drogga** [comment]
@garbarick - any progress..?

**Comment by garbarick** [comment]
Not yet. I have tried to compile app with new libmega.so and new java mega sdk which were extracted from official app. This path is failed. I have not time for going path described on https://github.com/meganz/sdk/tree/develop/examples/android/ExampleApp

**Comment by drogga** [comment]
In this case, you can add [Help Wanted] label \ tag to this issue, or [Won't Fix] if you give up on this, since I can't add it myself...

**Comment by garbarick** [comment]
fixed in 1.0.1
